### PR TITLE
CFY-7898: SSL becomes the default, with an option to disable via commandline

### PR DESCRIPTION
**Pull request by isaac-s**

### CODE REVIEW

**Comment by mcouthon** [comment]
Seems reasonable. But it'll probably break a bunch of our tests/build, so we'll wait until 4.4.

**Comment by mcouthon** [comment]
@isaac-s Before approving this, we need to consider the implications for our testing environments. You'd need to get the certificates from the manager somehow. I don't want to break absolutely every test with this PR without first having an idea of how we're going to fix it.

File: cfy_manager/main.py [hunk 1]
```diff
@@ -198,10 +206,12 @@ def _finish_configuration():
 @argh.arg('--private-ip', help=PRIVATE_IP_HELP_MSG)
 @argh.arg('--public-ip', help=PUBLIC_IP_HELP_MSG)
 @argh.arg('-a', '--admin-password', help=ADMIN_PASSWORD_HELP_MSG)
+@argh.arg('--no-ssl', action='store_true', help=NO_SSL_HELP_MSG)
```
Comment (mcouthon) [inline cfy_manager/main.py]: I'm not sure we need to do this via command line. I don't like the inputs getting clogged. If we're going with SSL by default, the user who want to go the no-ssl route can edit the config file (just a preference, you can disagree, of course)

**Comment by isaac-s** [comment]
@mcouthon I removed `--no-ssl`, as you requested.

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**